libs/types/src/permissions.rs

```diff
@@ -258,6 +263,7 @@ where
 				}
 				PoolRole::LoanAdmin => Ok(self.pool_admin.remove(PoolAdminRoles::RISK_ADMIN)),
 				PoolRole::TrancheInvestor(id, delta) => self.tranche_investor.remove(id, delta),
+				PoolRole::PODReadAccess => self.pool_admin.remove(PoolAdminRoles::POD_READ_ACCESS),
```

Comment:
Comparing to the rest of the implementations, it seems like this should be:
```suggestion
				PoolRole::PODReadAccess => Ok(self.pool_admin.remove(PoolAdminRoles::POD_READ_ACCESS)),
```

Reply:
Oh, sorry for missing this.